docs/releases/v2023.1.1.rst

```diff
@@ -9,6 +9,10 @@ Important notes
 
 Upgrades to this version are only supported from releases v2021.1 and later.
 
+**Note:**
+This release was found to be soft-bricking AVM Fritz!Box 7520 and 7530.
+We advice to not offer the release for these two devices until this get's fixed.
```

Comment:
```suggestion
We advice to not offer the release for these two devices until this gets fixed.
```

Reply:
good catch, I'll squash the commits once everyone's happy